packages/app/src/layouts/default.vue

```diff
@@ -24,7 +24,8 @@
       <BaseError
         v-if="query.data.value?.baseError"
         :gql="query.data.value.baseError"
-        :retry="resetErrorAndLoadConfig"
+        :retrying="retrying"
```

Comment:
It would be better to use `:retrying="mutation.loading.value"` here. If the button wasn't disabled, you could run into a problem with double clicking and resetting the loading state. Not a big deal since it is disabled, but we could get rid of the extra ref tracking the loading state.

Reply:
Good point - if we can drop `retrying` which seems to be 1:1 to `loading.value` that would be nice.